package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "test": "jest",
     "test:examples": "athloi run test --filter 'example-*'",
-    "build": "athloi run build --filter 'dotcom-*' --concurrency 10 --preserve-order",
-    "build:examples": "athloi run build --filter 'example-*' --concurrency 10",
+    "build": "tsc --build --verbose tsconfig.build.json && npm run build:browser --workspaces --if-present",
+    "build:examples": "npm run build --workspace=examples/ --if-present",
     "checktypes": "tsc --noEmit",
```

Comment:
**question:** I don't think we need line 11 (`checktypes`) now the `pre-commit` hook has been changed on line 91?

Reply:
Ahh I see you have done this in a later commit! 👍